src/includes/platforms/configuration/integrations/plugin.mdx

```diff
@@ -107,3 +107,21 @@ Available options:
 <PlatformContent includePath="configuration/reporting-observer" />
 
 </PlatformSection>
+
+<PlatformSection supported={["javascript"]}>
+
+### ContextLines
+
+_Import name: `Sentry.Integrations.ContextLines`_
+
+This integration attaches source code of inline JavaScript inside the page's HTML (e.g. JS in `<script>` tags) to errors captured by the SDK.
```

Comment:
```suggestion
This integration adds source file context to stack traces for captured errors from the inline JavaScript inside the page's HTML (e.g. JS in `<script>` tags).

```

Reply:
I tweaked this a little further but thanks for the suggestion!